specification/webdriver-classic.bs

```diff
@@ -0,0 +1,210 @@
+
+    <table class=simple>
+     <tr>
+      <td>POST</td>
+      <td>/session/{<var>session id</var>}/webextension</td>
+      <td>Load WebExtension</td>
+     </tr>
+
+     <tr>
+      <td>DELETE</td>
+      <td>/session/{<var>session id</var>}/webextension/{<var>extension id</var>}</td>
+      <td>Unload WebExtension</td>
+     </tr>
+    </table>
+  </section>
+
+  <section>
+    <h3>Errors</h3>
+    <table class=simple>
+     <tr>
+      <th>Error Code
+      <th>HTTP Status
+      <th>JSON Error Code
+      <th>Description
+     </tr>
+
+     <tr>
+      <td>no such web extension
+      <td>404
+      <td><code>no such web extension</code>
+      <td>No web extension matching the given extension id was found
+          amongst the <var>session</var>&apos;s loaded extensions.
+     </tr>
+
+      <tr>
+       <td>unable to load extension
+       <td>500
+       <td><code>unable to load extension</code>
+       <td>A command to load an extension could not be satisfied.
+     </tr>
+
+     <tr>
+       <td>unable to unload extension
+       <td>500
+       <td><code>unable to unload extension</code>
+       <td>A command to unload an extension could not be satisfied.
+     </tr>
+    </table>
+  </section>
+
+  <section>
+    <h3>Commands</h3>
+    <section>
+      <h2>WebExtensions</h2>
+      <p>The WebExtensions API provides an interface that allows
+      extensions to modify and enhance the capability of the browser.
+      This section describes the interaction with WebExtensions.
+
+      <section>
+        <h3>Load WebExtension</h3>
+        <table class="simple jsoncommand">
+         <tr>
+          <th>HTTP Method</th>
+          <th>URI Template</th>
+         </tr>
+         <tr>
+          <td>POST</td>
+          <td>/session/{<var>session id</var>}/webextension</td>
+         </tr>
+        </table>
+
+        <p>Based on the
+        <a href="https://www.w3.org/TR/webdriver-bidi/#command-webExtension-install">
+        BiDi implementation</a>, the remote end steps, given <var>session</var>,
+        <var>URL variables</var> and <var>parameters</var> are:
+
+        <ol>
+         <li><p>If <var>session</var>&apos;s current browsing context
+          is no longer open, return
+          <a href="https://w3c.github.io/webdriver/#dfn-error">error</a> with
+          <a href="https://w3c.github.io/webdriver/#dfn-error">error code</a>
+          no such window.
+
+          <li><p>If loading web extensions isn't supported, return
+          <a href="https://w3c.github.io/webdriver/#dfn-error">error</a>
+          with <a href="https://w3c.github.io/webdriver/#dfn-error">error code</a>
+           unsupported operation.
+          <li><p>Let <var>type hint</var> be the result of getting the
+          property "<code>type</code>" from <var>parameters</var>.
+            <ol>
+              <li type='a'><p> If <var>type hint</var> does not have the value of
+                      "path", "archivePath", or "base64", return
```

Comment:
Based on description, it was not clear if "hint" is a required or optional field.
This param could be optional i.e. if the value is not specified; or assume a default (say path). The name "hint" also seems like an optional param.

Reply:
the parameter is required and there is no default value if one isn't supplied